docs/pages/zkvm/sdk-benchmarking.mdx

```diff
@@ -0,0 +1,215 @@
+---
+title: "Nexus zkVM Benchmark"
+lang: en-US
+description: "Benchmark your Nexus project using the SDK"
+image: "/nexus-head.png"
+---
+
+import Image from "next/image";
+
+import { Steps } from "nextra/components";
+
```

Comment:
All the other pages start with
```
## Title of the page
```
So let's follow the pattern here too.

Reply:
updated in a61c6da

Reply:
In all the other pages, the we have only one `##` heading. The rest is `###` and more #'s.

Reply:
fixed in 88d1d92